tests/bin/distrib-name-dune-project/run.t

```diff
@@ -0,0 +1,32 @@
+When a package name is inferred from dune-project, it should only use the
+top-level one and ignore the ones under (package).
+
+  $ cat > dune-project << EOF
+  > (lang dune 2.7)
+  > (name first)
+  > (generate_opam_files true)
+  > (package
+  >  (name first))
+  > (package
+  >  (name second))
+  > EOF
+  $ cat > CHANGES.md << EOF
+  > # 1.0.0
+  > EOF
+  $ touch first.opam second.opam
+  $ git init 2> /dev/null . > /dev/null
+  $ git config user.name "dune-release-test"
+  $ git config user.email "pseudo@pseudo.invalid"
+  $ git add dune-project first.opam second.opam CHANGES.md
+  $ git commit -m 'Commit.' > /dev/null
+
+  $ dune-release tag --yes
+  [-] Extracting tag from first entry in CHANGES.md
+  [-] Using tag "1.0.0"
+  [+] Tagged HEAD with version 1.0.0
+
+  $ dune-release distrib --dry-run --skip-lint --skip-build --quiet
+  $ ls _build
+  first-1.0.0
+  first-1.0.0.build
+  first-1.0.0.tbz
```

Comment:
Could you add a test for

```
cat > dune-project << EOF
  > (lang dune 2.7)
  >  (name first)
```
To see what the semantics are if people hand-write these files and accidentally add whitespace?

Reply:
done!